src/views/template/topbar.html

```diff
@@ -1,31 +1,48 @@
-<div class="navbar-text" id="menu-toggle">
-    <span class="glyphicon glyphicon-menu-hamburger"></span>
-</div>
-<div>
-    <div class="navbar-text">
-        <div class="navbar-btn-group btn-group">
-            <button type="button" class="timer btn btn-default btn-xs active">
-                <span class="glyphicon glyphicon-pause"></span>
-            </button>
-            <button type="button" class="timer btn btn-default btn-xs">
-                <span class="glyphicon glyphicon-play"></span>
-            </button>
-        </div>
-        <span>
-            {{#l}}Day{{/l}}
-            <strong class="timer-days">0</strong>,
-            {{#l}}hour{{/l}}
-            <strong class="timer-hours">0</strong>
-        </span>
-    </div>
-</div>
-<div class="col-lg-offset-8">
-    <div class="navbar-text">
-        <strong>{{#l}}Satisfaction{{/l}}</strong>
-        <span id="demand-satisfaction">{{ satisfaction }}</span>%
+<div class="container-fluid">
+
+    <ul class="nav navbar-nav">
+        <li id="sidebar-left-toggle" style="cursor: pointer;">
+            <a>
+                <span class="glyphicon glyphicon-menu-hamburger"></span>
+                Menu
+            </a>
+        </li>
+    </ul>
+
+    <div class="btn-group navbar-left">
+        <button type="button" class="timer btn btn-default navbar-btn active">
+            <span class="glyphicon glyphicon-pause"></span>
+        </button>
+        <button type="button" class="timer btn btn-default navbar-btn">
+            <span class="glyphicon glyphicon-play"></span>
+        </button>
     </div>
-    <div class="navbar-text">
-        <strong>{{#l}}Money{{/l}}</strong>
-        <span id="money">{{ money }}</span>
+
+    <p class="navbar-text">
+        {{#l}}Day{{/l}}
+        <strong class="timer-days">0</strong>,
+        {{#l}}hour{{/l}}
+        <strong class="timer-hours">0</strong>
+    </p>
+
+    <ul class="nav navbar-nav navbar-right">
+        <li id="sidebar-right-toggle" style="cursor: pointer;">
+            <a>
+                <span class="glyphicon glyphicon-time"></span>
+                History
+            </a>
+        </li>
+    </ul>
+
+    <div class="nav navbar-nav navbar-right">
+        <p class="navbar-text">
+            <strong>{{#l}}Satisfaction{{/l}}</strong>
+            <span id="demand-satisfaction">{{ satisfaction }}</span>%
```

Comment:
I know this is not related at all, but could you change the ID into `satisfaction` instead of `demand-satisfaction`?